### PR TITLE
Move spam revision code to application controller

### PR DESCRIPTION
**Pull request by ananyo2012**
Make sure these boxes are checked before your pull request is ready to be reviewed and merged. Thanks!

* [x] tests pass -- `rake test`
* [x] code is in uniquely-named feature branch, and has been rebased on top of latest master (especially if you've been asked to make additional changes)
* [x] pull request are descriptively named
* [x] if possible, multiple commits squashed if they're smaller changes
* [ ] reviewed/confirmed/tested by another contributor or maintainer
* [x] `development.sqlite.example` has been updated if any database migrations were added

Please be sure you've reviewed our contribution guidelines at https://publiclab.org/wiki/contributing-to-public-lab-software

We have a loose schedule of reviewing and pulling in changes every Tuesday and Friday, and publishing changes on Fridays. Please alert developers on plots-dev@googlegroups.com when your request is ready or if you need assistance.

Thanks!

### CODE REVIEW

**Comment by ananyo2012** [comment]
@jywarren This fixes the issue that you have mentioned in https://github.com/publiclab/plots2/pull/566#issuecomment-225917860. Please review.

File: app/controllers/application_controller.rb [hunk 1]
```diff
@@ -118,4 +118,19 @@ def set_locale
       I18n.locale = params[:locale] || I18n.default_locale
     end
 
+  def spam_revision
```
Comment (jywarren) [inline app/controllers/application_controller.rb]: Thanks for this followup fix -- I think we need a more descriptive method name, though -- you should be able to see in the notes_controller and questions_controller that there may be a redirect, and that there may be alerts. It's long, but can you rename it "alert_and_redirect_moderated"?

**Comment by ananyo2012** [comment]
@jywarren Changed the method name. Please review.

**Comment by jywarren** [comment]
Great, is it rebased and tested? If so I can merge now (sorry, on my phone). Thanks!

**Comment by ananyo2012** [comment]
Please wait. Just rebasing. Actually there are alredy some tests for them
in notes controller and they are still passing.Isn't that should be enough?

ᐧ

**Comment by ananyo2012** [comment]
Rebased it. It can be merged now.
ᐧ

**Comment by jywarren** [comment]
Sure I just can't fun test from my phone so just asking if they're passing
for you.

Soon we'll have continuous integration tests that run on the server side
when a pull request is made, and that'll make this much faster! 😂

**Comment by ananyo2012** [comment]
Yeah that would be great!
ᐧ

**Comment by jywarren** [comment]
Done, thanks!

**Pull request merged**